src/ControllerV2/BeneficiaryController.php

```diff
@@ -3,16 +3,59 @@
 namespace App\ControllerV2;
 
 use App\Entity\Beneficiaire;
+use App\FormV2\FilterUser\FilterUserFormModel;
+use App\FormV2\FilterUser\FilterUserType;
 use App\FormV2\UserCreation\SecretQuestionType;
+use App\Repository\BeneficiaireRepository;
+use App\Repository\CentreRepository;
+use App\ServiceV2\PaginatorService;
 use Doctrine\ORM\EntityManagerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
-#[Route(path: '/beneficiary')]
+#[Route(path: '/beneficiaries')]
 class BeneficiaryController extends AbstractController
 {
+    #[Route(path: '', name: 'list_beneficiaries', methods: ['GET'])]
+    #[IsGranted('ROLE_MEMBRE')]
+    public function listBeneficiaries(
+        Request $request,
+        BeneficiaireRepository $repository,
+        PaginatorService $paginator,
+        CentreRepository $relayRepository,
+    ): Response {
+        $query = $request->query;
+        $formModel = new FilterUserFormModel(
+            $query->getAlpha('search'),
```

Comment:
Careful, getAlpha will remove some characters such as . for instance, and maybee -